### PR TITLE
Updated README.md, closes #985 #992

### LINKED ISSUE #985: Documentation Around Version Numbers
I believe some clarity around this would be good.  For example, I read that 19.4.1 should be used as per [here](http://www.openmage.org/magento-lts/install.html). 

However, when I place that on top of my magento code (just a zip file download and a copy over), in the admin panel I get Magento ver. 1.9.4.4.  

If I go to the github repo and just do git clone https://github.com/OpenMage/magento-lts.git  then copy the files over I get Magento ver. 1.9.4.5

I never see anything like 19.4.1.  Worse off, when I git clone https://github.com/OpenMage/magento-lts.git  I got commits from today.  I was under the assumption that a LTS version should have released versions and not just pull in all of the latest commits.

My recommendations here are as follows:

* Establish true documentation on how to upgrade / version numbers / etc.
* Add an OpenMage LTS version number in the footer of admin above or below current magento version. (Or replace completely, if allowed by liscense)

Maybe I am just confused, or perhaps this will confuse others as well.

Thanks!

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

- add branch alias to composer.json (see https://github.com/OpenMage/magento-lts/pull/992#issuecomment-638974892)
- add AFL 3.0 license
- removed duplicate LICENSE.html
- removed some git related text
- added some more info about 
  - new config options
  - new events
- updated install instructions (see #992) 

### Related Pull Requests

- OpenMage/magento-lts#992

### Fixed Issues (if relevant)

- Fixes OpenMage/magento-lts#985

### Questions or comments

- its WIP
- help wanted
- ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by kkrieger85 — DISMISSED** [review]
Yeah. Thx

File: README.md [hunk 1]
```diff
@@ -23,69 +23,93 @@ dependency definition. Each Magento `1.<minor>.<revision>` release will get its
...
-```json
-"openmage/magento-lts": "1.9.4.x"
 ```
+composer require openmage/magento-lts":"^19.4.0"
+```
+
+To get the latest changes use:
+
+```
+composer require openmage/magento-lts":"dev-master"
```
Comment (tmotyl) [inline README.md:45]: is dev-master correct here? we don't have master branch

**Review by tmotyl — DISMISSED** [review]

**Review by Flyingmana — CHANGES_REQUESTED** [review]
Please use something else than "dev-master"

Master is a deprecated term/phrase. Following githubs naming here, "dev-main" would be an appropriate alternative.

**Comment by colinmollenhour** [comment]
Looks good! No objections.

On the section for "If you want to contribute to the project:" what about mentioning the one-step dev environment by either adding a link to the [README](https://github.com/OpenMage/magento-lts/blob/1.9.4.x/dev/openmage/README.md) or adding the command to the list of shell commands given?

```
# Step 1: clone your fork
git clone https://github.com/<YOUR GIT USERNAME>/magento-lts
cd magento-lts
git remote add upstream https://github.com/OpenMage/magento-lts
git pull upstream 1.9.4.x

# Step 2: start the dev environment
dev/openmage/install.sh

# Step 3: hack some improvements and create a Pull Request
git checkout -b your-branch
git add -A
git commit
git push origin your-branch
```

File: README.md [hunk 2]
```diff
@@ -23,69 +23,93 @@ dependency definition. Each Magento `1.<minor>.<revision>` release will get its
...
 
-## Translations
+### Performance
+<small>ToDo: Please add performance related changes as run-time cache, ...</small>
 
-There are some new or changed tranlations, if you want add them to your locale pack please check:
+### New Config Options
+- `admin/design/use_legacy_theme`
+- `admin/emails/admin_notification_email_template`
+- `catalog/product_image/progressive_threshold`
```
Comment (sreichel) [inline README.md:76]:
```suggestion
- `admin/emails/admin_notification_email_template`
- `adminhtml_sales_order_create_save_before`
- `catalog/product_image/progressive_threshold`
```

Added new event #1168

File: README.md [hunk 3]
```diff
@@ -23,69 +23,93 @@ dependency definition. Each Magento `1.<minor>.<revision>` release will get its
...
 
-- `app/locale/en_US/*_LTS.csv`
+### New Events
+- `checkout_cart_product_add_before`
+- `sitemap_cms_pages_generating_before`
```
Comment (sreichel) [inline README.md:80]:
```suggestion
- 'adminhtml_sales_order_create_save_before'
- `checkout_cart_product_add_before`
- `sitemap_cms_pages_generating_before`
```
Added event #1168
Added #1168

**Comment by sreichel** [comment]
> what about mentioning the one-step dev environment

@colinmollenhour  Absolutely, but I'd like to leave that up to you. I would prefer the link to a README, this one you can manage yourself without needing PRs here.

**Review by colinmollenhour — APPROVED** [review]

**Review by kkrieger85 — APPROVED** [review]

**Review by Flyingmana — APPROVED** [review]

**Pull request merged**